pkg/serverconfig/types.go

```diff
@@ -68,6 +68,12 @@ type Customization struct {
 	CustomLogoFile       string `yaml:"customLogoFile,omitempty"`
 	// developerCatalog allows to configure the shown developer catalog categories.
 	DeveloperCatalog DeveloperConsoleCatalogCustomization `yaml:"developerCatalog,omitempty"`
+	ProjectAccess    ProjectAccess                        `yaml:"projectAccess,omitempty"`
+}
+
+// ProjectAccess contains options for project access roles
+type ProjectAccess struct {
+	AvailableClusterRoles []string `yaml:"availableClusterRoles,omitempty"`
```

Comment:
@rottencandy This "annotation" is required so that the stringified value starts with a lowercase character:

```suggestion
	AvailableClusterRoles []string `json:"availableClusterRoles,omitempty" yaml:"availableClusterRoles,omitempty"`
```

Reply:
Thanks I wasn't aware of this.